### PR TITLE
Create lookup table of placetype counts by country

### LINKED ISSUE #39: Create doc with placetype counts per country
Similar to [the placetypes by country table](https://github.com/whosonfirst/whosonfirst-cookbook/blob/master/definition/placetypes_by_country.md), it would be nice to have a sheet with per-country record counts per placetype.

I have the data (which can also be found browsing the Spelunker), it is just a matter of converting it to a markdown table and adding a new markdown file to this repo.

### PR DESCRIPTION
**Pull request by stepps00**
Fixes https://github.com/whosonfirst/whosonfirst-cookbook/issues/39

Creates a new lookup chart with placetype counts per country (based on per-country repo crawls). Doesn't include records with `"mz:is_current":0`, though I have those counts too, and can add to a separate chart if useful.

### CODE REVIEW

**Comment by nvkelso** [comment]
Looking good! 

1. To make it easier to interoperate with other datasets, can you add new columns for `iso3`, please? 
2. And then it's hard to know if the rows are `country` or `dependency` so consider adding a "placetype" column?

File: definition/placetype_counts_by_country.md [hunk 1]
```diff
@@ -0,0 +1,257 @@
+iso | country | macroregion | region | macrocounty | county | localadmin | locality | borough | macrohood | neighbourhood | microhood
+-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
```
Comment (nvkelso) [inline definition/placetype_counts_by_country.md]: If you add `:` to this 2nd column heading row it'll right align the table cells below:

> -- | -- | --: | --: | --: | --: | --: | --: | --: | --: | --: | --:

Comment (stepps00) [reply definition/placetype_counts_by_country.md]: Added in 1b1aa48. Let me know if this formatting works.

**Comment by stepps00** [comment]
https://github.com/whosonfirst/whosonfirst-cookbook/pull/40#issuecomment-671598747 taken care of in the most recent commits.

@nvkelso - take a look at the current chart formatting/data and let me know if anything else needs adjusting. I added in a "placetype" column for most repos.

**Comment by nvkelso** [comment]
Great to see the table changes. More questions:

- Looks like the French `dependency` aren't reporting correctly, I see blank cells in that placetype column.
- What are XX, XY, XZ? Why do those have "unknown" name? I think they mean something special and generic...
- In the same area of the scresnshot below, why are those missing `country` placetype?

image

**Comment by stepps00** [comment]
> Looks like the French dependency aren't reporting correctly, I see blank cells in that placetype column.

This table is based on placetype counts, per admin repo. In the French dependency cases, WOF classifies them at the `region` placetype. I only included `dependency` and `country` placetypes here, but I can adjust to include `region`s, too.

> What are XX, XY, XZ? Why do those have "unknown" name? I think they mean something special and generic...

These represent the placetype counts in each of the "unknown" placeholder repos. The best description for each is here: https://github.com/whosonfirst/whosonfirst-placetypes#iso-country-codes

> In the same area of the scresnshot below, why are those missing country placetype?

Good catch, I need to add `country` to the "Kosovo", "Null Island", and "Somaliland" rows.. I'll add in a new commit.

**Comment by stepps00** [comment]
I'm re-running the placetype counts to confirm data in this chart, then I'll merge.

**Pull request merged**